library/data/advancement/src/main/resources/quilt_advancement.mixins.json

```diff
@@ -3,7 +3,6 @@
   "package": "org.quiltmc.qsl.advancement.mixin",
```

Comment:
This whole file can be deleted and add a `noMixins()` to the build.gradle for the module